### PR TITLE
RGB separating script added

### PR DESCRIPTION
**Pull request by yelim237**
The script to separate RGB channels from a jpg file is added. Here is the output:

image

### CODE REVIEW

File: photo_120722/RGB.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import cv2
+
+image = cv2.imread('mosaic.jpg')
+
+red = image.copy()
+red[:, :, 0] = 0
+red[:, :, 1] = 0
```
Comment (hsmoon5458) [inline photo_120722/RGB.py:7]: Use numpy to change this value.

File: photo_120722/RGB.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+green = image.copy()
+green[:, :, 0] = 0
+green[:, :, 2] = 0
```
Comment (hsmoon5458) [inline photo_120722/RGB.py:11]: What I meant 'separation' is not change the color to r/g/b but extract the green value only from the image and show it.  That's why your image is so red, green and blue. Just extract the original value of r/g/b from the image and visualize it, not change's its rgb value on the image.

File: photo_120722/RGB.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+blue = image.copy()
+blue[:, :, 1] = 0
+blue[:, :, 2] = 0
+
+cv2.imshow('original', image)
+cv2.imshow('RED', red)
+cv2.imshow('GREEN', green)
+cv2.imshow('BLUE', blue)
+
+cv2.waitKey(0)
```
Comment (hsmoon5458) [inline photo_120722/RGB.py:22]: What is this for? Comment what you wrote.

File: photo_120722/RGB.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+blue = image.copy()
+blue[:, :, 1] = 0
+blue[:, :, 2] = 0
+
+cv2.imshow('original', image)
+cv2.imshow('RED', red)
+cv2.imshow('GREEN', green)
+cv2.imshow('BLUE', blue)
+
```
Comment (hsmoon5458) [inline photo_120722/RGB.py:21]:
```suggestion
# execute waitKey function to ~~~
```

File: photo_120722/RGB.py (hunk 1 above)
Comment (yelim237) [reply photo_120722/RGB.py:7]: okay

File: photo_120722/RGB.py (hunk 2 above)
Comment (yelim237) [reply photo_120722/RGB.py:11]: What I did here is to remove the color red and blue to make the image show the green color only regardless of the level of the green color. To rephrase what you said, I should extract one color (e.g., green) from the original image, not remove other colors to leave the color only, correct? I believe the result would be the same, but only the way to get the result might be different.

File: photo_120722/RGB.py (hunk 3 above)
Comment (yelim237) [reply photo_120722/RGB.py:22]: waitKey module lets the image stay on the screen until a user closes it out.

File: photo_120722/RGB.py (hunk 4 above)
Comment (yelim237) [reply photo_120722/RGB.py:21]: okay

Comment (hsmoon5458) [reply photo_120722/RGB.py:21]: Add comments for the rest of the code.

**Review by hsmoon5458 — APPROVED** [review]

**Pull request merged**